pkg/controller/plan/migration.go

```diff
@@ -632,30 +629,6 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			}
 		}
 		vm.Phase = r.next(vm.Phase)
-	case ScheduleVM:
-		step, found := vm.FindStep(r.step(vm))
-		if !found {
-			vm.AddError(fmt.Sprintf("Step '%s' not found", r.step(vm)))
-			break
-		}
-		waiting, rErr := r.waitingForFirstConsumer(vm)
-		if rErr != nil {
-			step.AddError(rErr.Error())
-			err = nil
-			break
-		}
-		err = r.setRunning(vm, waiting)
-		if err != nil {
-			step.AddError(err.Error())
-			err = nil
-			break
-		}
-		if !waiting {
-			vm.Phase = r.next(vm.Phase)
-			step.MarkCompleted()
-			step.Phase = Completed
-			step.Progress.Completed = step.Progress.Total
-		}
```

Comment:
yes that's possible. if we know the disk transfer finished, it would be nice to know we can run the vm before wrapping it up. but how you won't interfere the data/migration? it's tricky as written in your other comment.

Reply:
the tricky part I was referring to is if at some point in time X we find that we're able to start the VM but then it takes us 10 minutes let's say to complete pre-copy phase and when it's done, at some point in time Y, we shutdown the VM, transfer the last part and then find out that we can't run the VM on the destination.. but maybe we could just start the vm on the source again and treat this migration as a failed migration..

about what you wrote above, we should be able to know that the disk transfer(s) is done by the progress, no?

Reply:
once the CopyDisks(or Finalize in warm migration) is done we can say the disk transferred, or better after conversion (for vsphere).

Reply:
ok, we diverged :) so sounds like we can use that code but it seems better to drop it for now and add it back if we find it useful later on